Client.Linq/Internal/QueryExpressionTreeVisitor.cs

```diff
@@ -187,6 +187,88 @@ protected override Expression VisitMethodCall(MethodCallExpression expression)
                 return expression;
             }
 
+            if (expression.Method.DeclaringType == typeof(string))
+            {
+                if (expression.Method.Name.Equals("ToLower"))
+                {
+                    _context.QueryAggregator.AddImport("strings");
+                    var partsCount = _expressionParts.Count;
+                    Visit(expression.Object);
+                    var part = GetAndRemoveExpressionParts(partsCount, _expressionParts.Count);
+                    _expressionParts.Add(new StringFunction("toLower", part, null, null));
```

Comment:
Please create a `ToLowerFunction` class which will represent the `toLower` expression, instead of using the generic `StringFunction` class. This approach will enhance future maintenance and also facilitate contributions from other users.